typing/src/main/java/org/navgurukul/typingguru/utils/Utility.kt

```diff
@@ -80,19 +76,20 @@ object Utility {
             false
         }
     }
-
-    val alphabetList: ArrayList<String>
-        get() {
-            var c: Char
-            val result = ArrayList<String>()
-            c = 'a'
-            while (c <= 'z') {
-                result.add(c.toString())
-                ++c
+    companion object {
+        val TYPE_PRACTICE_TYPING = "practicetyping"
+        val alphabetList: ArrayList<String>
```

Comment:
This variable doesn't seem to be a candidate to be put inside companion object. Should be moved out of companion object. Also if the list is always the same, then better to assign is to a variable lazily instead of creating new list everytime